### DISCUSSION TITLE
Any support discord server or telegram channel

### DISCUSSION THREAD

**Discussion by killuazoldyckreal** [eternnoir/pyTelegramBotAPI#1651] [General]
Is there any support discord server or telegram channel of this library, if there is please provide its link.
Else please create one where users can ask their queries and helping volunteers can answer them.

**Comment by coder2020official** [comment]
This library has nothing to do with discord

**Reply by killuazoldyckreal** [reply]
> This library has nothing to do with discord

where did i said that it did, i asked for support channel whether it be on dc, tg or any other platform where helping volunteers can engage with users' queries. Cause here only few people could help who are maintaining this library as other coders who are willing to help doesn't get any notification on the queries for which they can help out. So basically I want to increase the helping hands here so the problems could be resovled swiftly and efficiently.

**Comment by coder2020official** [comment]
Hey! I got you, sorry for misunderstanding.
Here is the link for English group:
https://t.me/joinchat/Bn4ixj84FIZVkwhk2jag6A
We have russian & uzbek groups as well
